src/ui/public/vislib/lib/dispatch.js

```diff
@@ -310,5 +311,11 @@ export default function DispatchClass(Private) {
   }
 
 
+  function justifyOpacity(opacity) {
+    const decimalNumber = parseFloat(opacity, 10);
+    return (0 <= decimalNumber  && decimalNumber <= 1) ? decimalNumber : 0.5;
```

Comment:
Should we consider `Math.min(1, Math.max(0, decimalNumber))` instead? If I set an invalid value of `2` I'd probably rather have it be `1` than the `0.5` default.

If not, I'd move `0.5` to a variable like `const defaultOpacity = 0.5`.